src/element.toit

```diff
@@ -347,7 +347,9 @@ class Label extends Element implements ColoredElement:
     else if key == "alignment":
       alignment = value
     else if key == "label":
-      label = value
+      text = value
+    else if key == "text":
```

Comment:
Maybe do `key == "text" or key == "label"` instead? Start with the preferred and keep the equivalent old version as the second check.